src/dune_pkg/archive_driver.ml

```diff
@@ -0,0 +1,106 @@
+open Stdune
+module Process = Dune_engine.Process
+open Fiber.O
+
+module Command = struct
+  type t =
+    { bin : Path.t
+    ; args : archive:Path.t -> target_in_temp:Path.t -> string list
```

Comment:
What do you think about `gen_args`? I would expect `args` to be a fixed list not a function.

Reply:
I went with `make_args` as there's too much "gen" in dune already ;)